src/org/zaproxy/zap/extension/tokengen/TokenGenerator.java

```diff
@@ -58,16 +74,14 @@ protected Void doInBackground() throws Exception {
 			if (this.stopGenerating) {
 				break;
 			}
-
-			HttpMessage msg = this.httpMessage.cloneRequest();
-
+			HttpMessage msgClone = msg.cloneRequest();
+			Thread.sleep(51);

```

Comment:
Would you agree that using the "delay when fuzzing" variable is the way to go forward with this, instead of adding a separate value?